ui/app/pages/swaps/fee-card/__snapshots__/fee-card.test.js.snap

```diff
@@ -0,0 +1,175 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
From the official Jest site: "The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process."